maps/GWDM/2.0/GWDM/1.2/translation.jsonata

```diff
@@ -61,6 +65,21 @@ $fixObservedNode := function($obs){(
     }|
 )};
 
+$getDatasetLinkage := function() {
+  (
+    $datasetLinkage := {
+      "isDerivedFrom": $map($safeSplit(input.linkage.datasetLinkage.isDerivedFrom, ';,;'), function($v){{'title': $v}}),
```

Comment:
might need to be `[$map(...)]` as you have done in that function `$applyMap` 

JSONata is very annoying:
https://github.com/jsonata-js/jsonata/issues/218

definitely happens for `filter` but triple check for there when you have one item returned from the safeSplit

I doubt we'll ever need to use it though?

Reply:
Good point! Added (although I do expect we'll never use this mapping).